src/storage/table_heap.rs

```diff
@@ -51,7 +64,7 @@ impl TableHeap {
                 "tuple is too large, cannot insert"
             );
 
```

Comment:
`if there's no tuple in the page, and we can't insert the tuple, then this tuple is too large.`
can you put this comment to 
```
            assert!(
                last_table_page.num_tuples > 0,
                "tuple is too large, cannot insert"
            );
```